accounts/abi/bind/base.go

```diff
@@ -254,7 +254,7 @@ func (c *BoundContract) createDynamicTx(opts *TransactOpts, contract *common.Add
 	if gasFeeCap == nil {
 		gasFeeCap = new(big.Int).Add(
 			gasTipCap,
-			new(big.Int).Mul(head.BaseFee, big.NewInt(2)),
+			new(big.Int).Mul(head.BaseFee, big.NewInt(params.ElasticityMultiplier)),
```

Comment:
You'll need to import params, otherwise this is a build failure.

Reply:
I know without import params, it will be build failure. May I clarify with you that you intend to pass down the params all the way from the call stack?

Reply:
params is a package, `ElasticityMultiplier` is a global variable in it. You don't need to pass anything.

Reply:
Just add `"github.com/ethereum/go-ethereum/params"` to the import list and it will work. (If you configure your editor to run `goimports` on save, it would automatically add it).

Reply:
Attempted to update to use params. elasticityMultiplier but it seems params is not avaialble in this local context. Would it be your advice to add params all the way through the call-stack?